_posts/2020-10-01-intro-developer-advocate.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Hello I'm Brian, Presto Developer Advocate"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Hello, Presto nation!
+
+I wanted to take a moment to introduce myself! My name is Brian and I’m a new
+developer advocate working at Starburst. In this blog, I'll give you a little
+background on how I got here and cover what my role is in this community.
+
+![](/assets/blog/developer-advocate/brian.jpg)
+
+<!--more-->
+
+My career in computation and databases started in a previous life in the
+military. As luck would have it, I worked on a Big data team as my first job
+out of college! I was in a Hive shop that dealt with the typical outdated
+runtime and slow query turnaround. Eventually, our architect introduced our team
+to Presto as an alternative, and I worked with him to start testing and moving
+our existing use cases built on Hive to use Presto. We also used Elasticsearch
+and had a few cases that needed to perform joins and unions over the datasets in
+both Elasticsearch and Hive. There were a few use cases that were not going to
+immediately be transferable without some modification to the Elasticsearch
+connector. The first was [adding support for the Elasticsearch array 
```

Comment:
Maybe actually add a header before this sentence and move it to the next section ..